addons/godot_rl_agents/sensors/sensors_3d/GridSensor3D.gd

```diff
@@ -38,30 +38,47 @@ class_name GridSensor3D
 		cell_height = value
 		_update()	
 
-@export_range(1, 21, 2, "or_greater") var grid_size := 3:
-	get: return grid_size
+@export_range(1, 21, 2, "or_greater") var grid_size_x := 3:
+	get: return grid_size_x
 	set(value):
-		grid_size = value
+		grid_size_x = value
 		_update()
 
-var _obs_buffer : PackedByteArray
-var _box_shape : BoxShape3D
-var _collision_mapping : Dictionary
-var _n_layers_per_cell : int
+@export_range(1, 21, 2, "or_greater") var grid_size_z := 3:
+	get: return grid_size_z
+	set(value):
+		grid_size_z = value
+		_update()
+
+var _obs_buffer: PackedFloat64Array
```

Comment:
There was an error from Python when starting training:
`TypeError: can't convert np.ndarray of type numpy.str_. The only supported types are: float64, float32, float16, complex64, complex128, int64, int32, int16, int8, uint8, and bool.
exit was not clean, using atexit to close env`

Using float64 fixed this.